frame/reward-curve/src/lib.rs

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This file is part of Kulupu.
+//
+// Copyright (c) 2020 Wei Tang.
```

Comment:
```suggestion
// Copyright (c) 2020 Shawn Tabrizi.
```

In Kulupu, each contributor retains their copyright themselves, as we sign no CLA. Should write the line as you, and then when other contributors or me edit this file majorly, we then add our lines there as well.